_tests/managed_tests/tests/__snapshots__/PhpcompatibilityTest__test_phpcompatibility_sut_81_to_82_woo_php_wp_6ee042538e1935736a9856e687d0c150__1.php

```diff
@@ -8,8 +8,8 @@
             "wordpress_version": "6.0.0-normalized",
             "woocommerce_version": "6.0.0-normalized",
             "php_version": "7.4",
-            "max_php_version": "8.4",
-            "min_php_version": "7.4",
```

Comment:
Also not sure why this changed, but it looks correct as per the test file name

Reply:
Same as https://github.com/woocommerce/qit-cli/pull/367/files/6631a012d9ad57c831fc48cb779686d35a2c92de#r2466971831